crates/astria-cli/src/cli/rollup.rs

```diff
@@ -100,6 +102,17 @@ pub struct ConfigCreateArgs {
         default_value = DEFAULT_SEQUENCER_RPC
     )]
     pub sequencer_rpc: String,
+    /// Optional. Will default to 'localdev.me' for local deployments. Will need to separately
+    /// configure other hosts
+    #[clap(
+        long = "hostname", 
+        env = "ROLLUP_HOSTNAME", 
+        default_value = DEFAULT_HOSTNAME
+    )]
+    pub hostname: String,
+    /// Configures the k8s namespace rollup will be deployed to
+    #[clap(long, env = "ROLLUP_NAMESPACE", default_value = DEFAULT_NAMESPACE)]
+    pub(crate) namespace: String,
```

Comment:
why is this `pub(crate)` but everything else is `pub`?

Reply:
I moved it from a different config and missed. Updating.